packages/server/utils/stripe/index.ts

```diff
@@ -0,0 +1,6 @@
+import StripeManager from './StripeManager'
+import StubStripeManager from './StubStripeManager'
+
+export function getStripeManager(): StripeManager {
+  return process.env.CI ? StubStripeManager() : new StripeManager()
```

Comment:
I'm not sure about the reliability of this check. Due to the fact that it looks too unreliable to prevent a possible problem in production....

I'm thinking about using NODE_ENV === 'test' or introducing a new env variable `STRIPE_MODE` which by default (even if it's missing) should be interpreted as the real stripe system.